x/model/types/messages_model_test.go

```diff
@@ -498,6 +530,20 @@ func TestMsgCreateModel_ValidateBasic(t *testing.T) {
 				return msg
 			}(validMsgCreateModel()),
 		},
+		{
+			name: "LsfUrl is omitted",
```

Comment:
I think we need to add the same cases (negative and positive) for `LsfUrl` and `LsfRevision` for `UpdateModel`